edx_proctoring/backends/tests/test_rest.py

```diff
@@ -182,6 +182,7 @@ def test_on_review_callback(self):
         self.assertEqual(payload, new_payload)
 
     def test_get_javascript(self):
-        # A real backend would return real javascript from backend.js
+        # A real backend would return a real bundle url from webpack
+        # but in this context we'll fail looking up webpack's stats file
         with self.assertRaises(IOError):
```

Comment:
If you changed the code to return an empty string if there's no package, then the code will not raise IOError, in which case, this test should fail. Does it fail?

Reply:
There are two files loaded when looking up a bundle: first the library looks up the stats file (which in this test case, fails with an IOError). If that were to succeed, then we'd look up the bundle's location from inside the stats file. In the case that the bundle's name isn't found in the stats file, get_files will return an empty list.

I hardened the code and added a test illustrating this, but the test we're talking about here does pass.